homework1/src/ChangeMinElement.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Arrays;
+import java.util.Collections;
+
+public class ChangeMinElement {
+    public static void main(String[] args) {
+        Integer[] numbers = {7, 10, 11, 15, 3, 3, 12, 10, 11, 20};
+        System.out.print("Original array: " + Arrays.toString(numbers));
+
+        int min = Collections.min(Arrays.asList(numbers));
+        System.out.println("\nFind min number of array: " + min);
+
+        int total = 0;
+        int min_duplicates_length = 0;
+        for (int position = 0; position < numbers.length; position++) {
```

Comment:
same
prefer to use variable name as "i" in simple loops

Reply:
Fixed in commit: f1536a4